airbyte-integrations/connectors/destination-dev-null/metadata.yaml

```diff
@@ -1,8 +1,8 @@
 data:
-  connectorSubtype: file
+  connectorSubtype: unknown
```

Comment:
note: double-check how @stephane-airbyte resolved this (and also make sure our versions align)

Reply:
(current version is 1.20.1. Please bump it)